### PR TITLE
Feat ui fixes

### PR DESCRIPTION
**Pull request by rishi9684**
#Mobile Fixes
#Login Box with Mobile UI

### CODE REVIEW

File: utility/index.js [hunk 1]
```diff
@@ -11,13 +11,15 @@ export const templateLoaderMixin = {
       return (gridTemplatePath) => importTemplate(`${gridTemplatePath}`)
     },
     templateFolderName() {
+      debugger
```
Comment (SukantGujar) [inline utility/index.js:14]: Remove all debuggers

**Review by SukantGujar — CHANGES_REQUESTED** [review]

File: utility/index.js (hunk 1 above)
Comment (rishi9684) [reply utility/index.js]: Removed

File: nuxt.config.js [hunk 2]
```diff
@@ -107,7 +107,7 @@ export default {
     apiEndpoint: '/svc/api/',
     backendBaseUrl: process.env.PUBLIC_DOMAIN || '',
     baseURL: `https://${process.env.PUBLIC_DOMAIN}${basePath}`,
-    eventUrl: process.env.GETEVENTURL || 'event.test.bitpod.io',    
+    eventUrl: process.env.GETEVENTURL || 'event.test.bitpod.io',
```
Comment (SukantGujar) [inline nuxt.config.js:110]: GET_EVENT_URL

**Review by SukantGujar — CHANGES_REQUESTED** [review]

File: nuxt.config.js (hunk 2 above)
Comment (rishi9684) [reply nuxt.config.js]: changed

**Review by SukantGujar — APPROVED** [review]

**Review by SukantGujar — APPROVED** [review]

**Pull request merged**